src/app/ui/components/containers/dialog/dialog.tsx

```diff
@@ -53,6 +53,7 @@ export function Dialog({
             position: 'fixed',
             inset: 0,
             animation: 'overlayShow 150ms cubic-bezier(0.16, 1, 0.3, 1)',
+            zIndex: 999,
```

Comment:
I need to add a high z-index here, otherwise sometimes other content appears above the dialog. Namely - tab and ordinal iframe

![Screenshot 2024-06-20 at 10 30 01](https://github.com/leather-wallet/extension/assets/2938440/8c0c73d9-b92e-4356-a919-a46623afb5cf)
![Screenshot 2024-06-20 at 10 25 02](https://github.com/leather-wallet/extension/assets/2938440/6773adc0-8ddb-4455-8303-6cb63487bf4d)